src/commands/queue.ts

```diff
@@ -50,7 +50,28 @@ export function addQueueCommands(bot: DiscordBot, config: BotConfig, daos: Daos)
         await message.reply(`Failed to add anything to the queue.`);
         console.log(e);
       }
-    }
+    },
+
+    'replace': async (message: Message, params: string[]) => {
+      const queuePlayer = queuePlayerManager.get(message.guild.id);
+      if (queuePlayer.queuedTracks !== 0) {
+        await queuePlayer.clear();
+        await message.reply('The queue is already empty though...');
```

Comment:
If the queue is already empty that's okay I'd prefer no unnecessary messaging from the bot :)

Reply:
I intended to reply "I have cleared the queue", but copied the wrong line apparently.